Packs/FireEyeHelix/Integrations/FireEyeHelix/FireEyeHelix.py

```diff
@@ -1614,7 +1614,11 @@ def search_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
     Returns:
         Outputs
     """
-    query = build_mql_query(**args)
+    query = build_mql_query(
+        query=args.get('query', ''), start=args.get('start'), end=args.get('end'), page_size=args.get('page_size'),
+        limit=args.get('limit'), offset=args.get('offset'), groupby=args.get('groupby'), sort_by=args.get('sort_by'),
+        sort_order=args.get('sort_order')
+    )
```

Comment:
looks a bit messy to me. Also, is this the only place this fix is required? what about the other function calls?

Reply:
what do you mean messy? do you a suggestion here?

went over the other functions, it seems to be the only place to edit for 2 reasons:
1. other commands that have `headers` arg don't pass `**args` as in the search command
2. other commands that pass `**args` dont have the `headers` command

also, we dont have an instance so i prefer to do only required changes to fix the issue

Reply:
just declare the arguments outside I think

Reply:
i guess its matter of code style, but i really dont see how that makes it "less messy"